src/vs/workbench/api/node/extHostTypeConverters.ts

```diff
@@ -405,11 +405,19 @@ export namespace Suggest {
 	}
 };
 
+
+function convertSignatureDocumentation(documentation: string | types.MarkdownString): undefined | string | htmlContent.IMarkdownString {
+	if (!documentation) {
+		return undefined;
+	}
+	return typeof documentation === 'string' ? documentation : MarkdownString.from(documentation);
+}
+
```

Comment:
lgtm but should be moved into the `MarkdownString` namespace with a better name, like `fromStrict` signaling that is just strings or the MarkdownString, not the MarkedString. Or rename the `from` function to something like `fromOldOrNew`